geomstats/learning/intrinsic_dimension_estimation.py

```diff
@@ -0,0 +1,100 @@
+"""Intrinsic Dimension Estimation."""
+
+import geomstats.backend as gs
+
+
+class LevinaBickelEstimator:
```

Comment:
Excellent 🎉 !!

Reply:
I think `IntrinsicDimensionEstimator` would be more intuitive: instead of giving it a name, with tell *what* it is. 

NIT: The file could also be renamed to `intrinsic_dimension_estimator.py`, for consistency.

Any reason why this does not inherit from `BaseEstimator`? Maybe inheriting from BaseEstimator restricts the gs.arrays that we can use (i.e. since sklearn mostly works in np.arrays, maybe we can't use pytorch arrays there?)

Reply:
it would be intutive. but they are many